opentelemetry/benches/metrics.rs

```diff
@@ -1,4 +1,4 @@
-use criterion::{criterion_group, criterion_main, Criterion};
+use criterion::{black_box, criterion_group, criterion_main, BatchSize, Criterion};
```

Comment:
could you add the numbers here, similar to https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-sdk/benches/metric_counter.rs#L1-L13 for  easy reference.

Reply:
Updated.